docs/excel/excel-add-ins-ranges.md

```diff
@@ -526,21 +526,47 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
-**Data in range before number format is set**
+#### Data in range before number format is set
 
 ![Data in Excel before format is set](../images/excel-ranges-format-font-and-fill.png)
 
-**Data in range after number format is set**
+#### Data in range after number format is set
 
 ![Data in Excel after format is set](../images/excel-ranges-format-numbers.png)
 
+## Read or write to an unbounded range
+
+### Read an unbounded range
+
+An unbounded range address is a range address that specifies either entire column(s) or entire row(s). For example:
+
+- Range addresses comprised of entire column(s):<ul><li>`C:C`</li><li>`A:F`</li></ul>
+- Range addresses comprised of entire row(s):<ul><li>`2:2`</li><li>`1:4`</li></ul>
+
+When the API makes a request to retrieve an unbounded range (for example, `getRange('C:C')`), the response will contain `null` values for cell-level properties such as `values`, `text`, `numberFormat`, and `formula`. Other properties of the range, such as `address` and `cellCount`, will contain valid values for the unbounded range.
+
+### Write to an unbounded range
+
+You cannot set cell-level properties such as `values`, `numberFormat`, and `formula` on an unbounded range because the input request is too large. For example, the following code snippet is not valid because it attempts to specify `values` for an unbounded range. The API will return an error if you attempt to set cell-level properties for an unbounded range.
```

Comment:
```suggestion
You cannot set cell-level properties such as `values`, `numberFormat`, and `formula` on an unbounded range because the input request is too large. For example, the following code snippet is not valid because it attempts to specify `values` for an unbounded range. The API returns an error if you attempt to set cell-level properties for an unbounded range.
```